modules/installation-guide/partials/proc_run-several-workspaces-at-a-time.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// advanced-configuration
+
+[id="run-several-workspaces-at-a-time_{context}"]
+= Running more than one workspace at a time
+
+This procedure describes how to run more than one workspace simultaneously, allowing a user to perform several tasks at once.
+
+.Prerequisites
+
+* The `{platforms-cli}` tool is available.
+* An instance of {prod-short} running in {platforms-name}.
++
+[NOTE]
+====
+The following commands use the default {platforms-namespace}, `{prod-namespace}`, as a user's example for the `-n` option.
+====
+
+.Procedure
+
+. Remove the limit of concurrently running workspaces per user:
+
+* For Operator deployments:
++
+[subs="+quotes,+attributes"]
+----
+$ {orch-cli} patch checluster {prod-checluster} -n {prod-namespace} --type merge \
+-p '{ "spec": { "server": {"customCheProperties": {"CHE_LIMITS_USER_WORKSPACE_RUN_COUNT": "-1"} } }}'
+----
+ifeval::["{project-context}" == "che"]
+* For Helm Chart deployments:
++
+[subs="+quotes,+attributes"]
+----
+$ helm upgrade che -n {prod-namespace} --set global.workspace.number=-1
+----
+endif::[]
+
+. Set the `per-workspace` or `unique` PVC strategy. See, xref:administration-guide:che-workspaces-architecture.adoc#configuring-a-{prod-id-short}-workspace-with-a-persistent-volume-strategy_{prod-id-short}-workspace-configuration[Configuring a {prod-short} workspace with a persistent volume strategy].
++
+[NOTE]
+====
+To use `common` PVC strategy, configure the persistent volumes to use the `ReadWriteMany` access mode.
```

Comment:
Might be better to phrase this as "If you choose to use the `common` PVC strategy instead of one of the recommended ones above, it's advisable to configure the persistent volumes to use the `ReadWriteMany` access mode so that your common PVC can be read from and written to from any of your concurrent workspaces." ?

Reply:
====
With the `common` PVC strategy, configure the persistent volumes to use the `ReadWriteMany` access mode so that the common PVC can be read from and written to from any of the user's concurrent workspaces.
====